frontend/src/components/eMIB/constants.js

```diff
@@ -44,7 +44,23 @@ export const actionShape = PropTypes.shape({
   reasonsForAction: PropTypes.string,
   task: PropTypes.string,
   emailType: PropTypes.oneOf(Object.keys(EMAIL_TYPE)),
-  emailTo: PropTypes.string,
-  emailCc: PropTypes.string,
+  emailTo: PropTypes.arrayOf(PropTypes.number),
+  emailCc: PropTypes.arrayOf(PropTypes.number),
   emailBody: PropTypes.string
 });
+
+// The stucture of a contact in the addressbook
+export const contactShape = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  role: PropTypes.string.isRequired
+});
+
+// The stucture of a option, including
+// a visible display name (text) is created in transformations.js from contactShape's name and role
+// the value is the same as the transformed contactShape's id
+// this is used for a drop-down menu
+export const optionShape = PropTypes.shape({
```

Comment:
I don't see this imported or used anywhere?

Reply:
dropped in  72b1385